srcpkgs/lxcfs-devel

```diff
@@ -0,0 +1 @@
+lxcfs/
```

Comment:
remove the trailing slash

Reply:
Gonna try and test and see if it does. I didn't know anything about valgrind, so thanks for pointing that out. 

How do I remove the trailing slash on a symlink'd dir?

Reply:
Just don't include it when making the symlink. Your shell might autocomplete it but just put lxcfs.

Reply:
@CameronNemo https://paste.pound-python.org/show/ZibDAQWOkG58BfuIPVF1/ 
If I am not mis-reading it, looks like one may be left ?